meta-oe/recipes-support/avro/avro-c++_1.11.3.bb

```diff
@@ -18,3 +18,12 @@ SRC_URI = "git://github.com/apache/avro;branch=${BRANCH};protocol=https \
 S = "${WORKDIR}/git/lang/c++"
 
 inherit cmake pkgconfig
+
+BBCLASSEXTEND = "native nativesdk"
+
+inherit multilib_script
+MULTILIB_SCRIPTS = "\
+    ${PN}-dev:${libdir}/libavrocpp.so \
+    ${PN}-dev:${libdir}/libavrocpp.so.1.11.3 \
+    ${PN}-dev:${libdir}/cmake \
+    "
```

Comment:
Why is this needed? I don't think so as libdir value will be different, so they shouldn't conflict.

Reply:
I removed it